src/views/RepoDetailIssueView/IssueChart.tsx

```diff
@@ -146,13 +150,21 @@ const IssueChart: React.FC<IssueChartProps> = (props) => {
     const instance = echarts.getInstanceByDom(chartDOM as any);
     if (instance) {
       instance.setOption(option);
+      if (onClick) {
+        instance.on('click', (params) => {
+          onClick(issueclickparams, params);
+        });
+      }
     }
   }, []);
 
   return <div ref={divEL} style={{ width, height }}></div>;
 };
 
+let issueclickparams: { data: number[]; marker: any; seriesName: any };
+
 const tooltipFormatter = (params: any) => {
+  issueclickparams = params[0];
```

Comment:
So this line will be `curMonth = params[0].data[0];`

Note: in JavaScript variables are conventionally named in [lowerCamelCase](https://www.techtarget.com/whatis/definition/lowerCamelCase).